org-formation/600-access/_tasks.yaml

```diff
@@ -388,3 +388,31 @@ SynapseAthenaUserAccessPolicy:
         ]
       }
     PolicyName: SynapseAthenaUserAccessPolicy
+
+SynapseLlmBedrockAgentPolicy:
```

Comment:
@zaro0508 , wondering if I should have the policy here or if I should move it as a resource alongside the bedrock agent role below (org-formation/745-bedrock-agent-role/bedrock-agent-role.yaml).

Reply:
> @zaro0508 , wondering if I should have the policy here or if I should move it as a resource alongside the bedrock agent role below.

You'll notice that existing policy definitions created in here are meant to be applied to all accounts (`Account: '*'`) because some AWS features require us to apply the same policy.  If you are intending to apply this to one account then I would say that it's more appropriate to  setup this policy with the resource that needs it.